src/components/Viewer/Panel/Qualification.jsx

```diff
@@ -12,46 +13,81 @@ import en from '../locales/fr.json'
 
 const locales = { fr, en }
 
-const Qualification = ({ file, t }) => {
-  const { name, metadata } = file || {}
-  const qualificationLabel = metadata?.qualification.label
+const validPageLabel = page => page === 'front' || page === 'back'
+
+const Qualification = ({ file, t, f, lang }) => {
+  const scannerT = getBoundT(lang)
+  const client = useClient()
+  const [currentUser, setCurrentUser] = useState(null)
+
+  const { name: filename, metadata } = file || {}
+  const { qualification } = metadata || {}
+  const { label, page: pageLabel, featureDate } = qualification || {}
+
+  // TODO Improve it when other contact choices are needed
+  useEffect(() => {
+    let isMounted = true
```

Comment:
Maybe we could share the isMounted in a `utils` file, we find again this code [here](https://github.com/cozy/mespapiers/pull/56/commits/d1a4d3d251cadd39b5e6fd6f1e1897f427fb0326#diff-d13c6ecb8e3e7e8404d05f4cf7e72c6027c70a80f3052241dc61df7a632566d6R28)

Reply:
Effectivement, il est même utilisé dans d'autres endroits
Je verrais ça dans une autre PR dédiée 👍